src/utils/sprint-utils.ts

```diff
@@ -0,0 +1,103 @@
+import { Person } from "src/generated/client";
+import { Allocations, Projects, Tasks } from "src/generated/homeLambdasClient";
+
+/**
+ * Retrieve total time entries for an Allocation
+ * 
+ * @param allocation task allocated within a project
```

Comment:
Allocations are not really related to tasks. Perhaps just allocation Allocation here would be fine to indicate the name is allocation and its type is Allocation?
Similar change can be made for the other param comments